task/orka-modular/0.1/README.md

```diff
@@ -0,0 +1,257 @@
+# Run macOS Builds with Tekton and Orka by MacStadium
+
+These `Tasks` are for utilizing multiple macOS build agents, in parallel or in series, in your [Orka environment](https://orkadocs.macstadium.com).
+
+## `orka-modular`
+
+`orka-modular.yaml` combines the following three modular `Tasks`: `orka-init`, `orka-deploy`, and `orka-teardown`.
+
+With these `Tasks`, you can build an end-to-end `Pipeline` that works with multiple macOS build agents.
+
+### `orka-init`
+
+A `Task` that creates a VM template with the specified configuration. All operations in this `Task` are performed against an Orka environment.
+
+### `orka-deploy`
+
+A `Task` that deploys a VM instance from a specified VM template. Usually, you would use the VM template created with `orka-init`. 
+
+### `orka-teardown`
+
+A `Task` that cleans up the Orka environment after your workload completes. This `Task` the VM instances deployed with `orka-deploy` and their related templates.
```

Comment:
Second sentence here - "This Task cleans up the VM instances ..." or something like that, just reads a bit awkwardly